<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoodChart</class>
 <widget class="QWidget" name="MoodChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 整体滚动条背景 */
QScrollBar:vertical {
    background: #eaffea;  /* 浅绿色滚动条槽背景 */
    width: 12px;
    margin: 0px 0px 0px 0px;
    border-radius: 6px;
}

/* 滚动条滑块 */
QScrollBar::handle:vertical {
    background: #a0e0a0; /* 深一点的绿色 */
    min-height: 20px;
    border-radius: 6px;
}

/* 滚动条滑块hover状态 */
QScrollBar::handle:vertical:hover {
    background: #7ac97a;
}

/* 滚动条两端箭头 */
QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical {
    height: 0px; /* 不显示箭头 */
}

/* 滚动条槽 */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>727</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>1200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 整体滚动条背景 */
QScrollBar:vertical {
    background: #eaffea;  /* 浅绿色滚动条槽背景 */
    width: 12px;
    margin: 0px 0px 0px 0px;
    border-radius: 6px;
}

/* 滚动条滑块 */
QScrollBar::handle:vertical {
    background: #a0e0a0; /* 深一点的绿色 */
    min-height: 20px;
    border-radius: 6px;
}

/* 滚动条滑块hover状态 */
QScrollBar::handle:vertical:hover {
    background: #7ac97a;
}

/* 滚动条两端箭头 */
QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical {
    height: 0px; /* 不显示箭头 */
}

/* 滚动条槽 */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
    </property>
    <widget class="QGroupBox" name="weeklyGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>715</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>近 7 天心情趋势</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>711</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="weeklyChartLayout"/>
     </widget>
    </widget>
    <widget class="QGroupBox" name="monthlyGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>710</y>
       <width>715</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>月度心情趋势</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>711</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="monthlyChartLayout"/>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Songti TC</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;心情统计分析&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本月的心情指数是：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本月出现最多的心情是：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本周的心情指数是：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本周出现最多的心情是：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本周写日记次数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Songti SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>你本月写日记次数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="weekIconLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="weekEmojiLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>58</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
    <widget class="QLabel" name="weekDairyTime">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>58</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="monthIconLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="monthDairyTime">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="monthEmojiLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0)</string>
     </property>
     <property name="text">
      <string>(最高5分，默认3分)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>441</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>月度表情使用情况</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>20</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="monthlyEmojiLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
